commit c6632416fc4f55f21caa51937b5207bb8eb5a63f
Author: sveinung <sveinung@a0f10bec-cc02-0410-94fc-a9cfff90b4cd>
Date:   Wed Oct 28 09:25:17 2015 +0000

    Europe 1901: make all players AI controlled
    
    so the scenario will work better "out of the box" for single or few player
    games.
    
    See patch #6493
    
    git-svn-id: svn://svn.gna.org/svn/freeciv/trunk@30267 a0f10bec-cc02-0410-94fc-a9cfff90b4cd

diff --git a/data/scenarios/europe_1901.sav b/data/scenarios/europe_1901.sav
index 152e9c6..c04a478 100644
--- a/data/scenarios/europe_1901.sav
+++ b/data/scenarios/europe_1901.sav
@@ -1744,7 +1744,7 @@ government_name="Monarchy"
 city_style_by_name="Classical"
 is_male=TRUE
 is_alive=TRUE
-ai.control=FALSE
+ai.control=TRUE
 diplstate={"type","max_state","first_contact_turn","turns_left","has_reason_to_cancel","contact_turns_left","embassy","gives_shared_vision"
 5,5,0,0,0,0,FALSE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
@@ -3394,7 +3394,7 @@ government_name="Monarchy"
 city_style_by_name="European"
 is_male=TRUE
 is_alive=TRUE
-ai.control=FALSE
+ai.control=TRUE
 diplstate={"type","max_state","first_contact_turn","turns_left","has_reason_to_cancel","contact_turns_left","embassy","gives_shared_vision"
 2,5,0,16,0,20,TRUE,FALSE
 5,5,0,0,0,0,FALSE,FALSE
@@ -5163,7 +5163,7 @@ government_name="Monarchy"
 city_style_by_name="European"
 is_male=TRUE
 is_alive=TRUE
-ai.control=FALSE
+ai.control=TRUE
 diplstate={"type","max_state","first_contact_turn","turns_left","has_reason_to_cancel","contact_turns_left","embassy","gives_shared_vision"
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
@@ -6860,7 +6860,7 @@ government_name="Republic"
 city_style_by_name="Celtic"
 is_male=TRUE
 is_alive=TRUE
-ai.control=FALSE
+ai.control=TRUE
 diplstate={"type","max_state","first_contact_turn","turns_left","has_reason_to_cancel","contact_turns_left","embassy","gives_shared_vision"
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
@@ -8618,7 +8618,7 @@ government_name="Monarchy"
 city_style_by_name="European"
 is_male=TRUE
 is_alive=TRUE
-ai.control=FALSE
+ai.control=TRUE
 diplstate={"type","max_state","first_contact_turn","turns_left","has_reason_to_cancel","contact_turns_left","embassy","gives_shared_vision"
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
@@ -10452,7 +10452,7 @@ government_name="Monarchy"
 city_style_by_name="Classical"
 is_male=TRUE
 is_alive=TRUE
-ai.control=FALSE
+ai.control=TRUE
 diplstate={"type","max_state","first_contact_turn","turns_left","has_reason_to_cancel","contact_turns_left","embassy","gives_shared_vision"
 2,5,0,16,0,20,TRUE,FALSE
 4,4,0,0,0,20,TRUE,FALSE
@@ -12089,7 +12089,7 @@ government_name="Monarchy"
 city_style_by_name="European"
 is_male=TRUE
 is_alive=TRUE
-ai.control=FALSE
+ai.control=TRUE
 diplstate={"type","max_state","first_contact_turn","turns_left","has_reason_to_cancel","contact_turns_left","embassy","gives_shared_vision"
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
@@ -13744,7 +13744,7 @@ government_name="Monarchy"
 city_style_by_name="Classical"
 is_male=TRUE
 is_alive=TRUE
-ai.control=FALSE
+ai.control=TRUE
 diplstate={"type","max_state","first_contact_turn","turns_left","has_reason_to_cancel","contact_turns_left","embassy","gives_shared_vision"
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
@@ -15360,7 +15360,7 @@ government_name="Monarchy"
 city_style_by_name="Classical"
 is_male=TRUE
 is_alive=TRUE
-ai.control=FALSE
+ai.control=TRUE
 diplstate={"type","max_state","first_contact_turn","turns_left","has_reason_to_cancel","contact_turns_left","embassy","gives_shared_vision"
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
@@ -16936,7 +16936,7 @@ government_name="Monarchy"
 city_style_by_name="European"
 is_male=TRUE
 is_alive=TRUE
-ai.control=FALSE
+ai.control=TRUE
 diplstate={"type","max_state","first_contact_turn","turns_left","has_reason_to_cancel","contact_turns_left","embassy","gives_shared_vision"
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
@@ -18539,7 +18539,7 @@ government_name="Monarchy"
 city_style_by_name="European"
 is_male=TRUE
 is_alive=TRUE
-ai.control=FALSE
+ai.control=TRUE
 diplstate={"type","max_state","first_contact_turn","turns_left","has_reason_to_cancel","contact_turns_left","embassy","gives_shared_vision"
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
@@ -20137,7 +20137,7 @@ government_name="Republic"
 city_style_by_name="European"
 is_male=TRUE
 is_alive=TRUE
-ai.control=FALSE
+ai.control=TRUE
 diplstate={"type","max_state","first_contact_turn","turns_left","has_reason_to_cancel","contact_turns_left","embassy","gives_shared_vision"
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
@@ -20207,7 +20207,7 @@ ai={"spam","countdown","war_reason","patience","warn_space","ask_peace","ask_all
 0,-1,5,0,0,0,0,0
 1,-1,5,0,0,0,0,0
 }
-ai.skill_level=0
+ai.skill_level=7
 ai.is_barbarian=0
 gold=1520
 rates.tax=40
@@ -21700,7 +21700,7 @@ government_name="Monarchy"
 city_style_by_name="Celtic"
 is_male=TRUE
 is_alive=TRUE
-ai.control=FALSE
+ai.control=TRUE
 diplstate={"type","max_state","first_contact_turn","turns_left","has_reason_to_cancel","contact_turns_left","embassy","gives_shared_vision"
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
@@ -21770,7 +21770,7 @@ ai={"spam","countdown","war_reason","patience","warn_space","ask_peace","ask_all
 1,-1,5,0,0,0,0,0
 2,-1,5,0,0,0,0,0
 }
-ai.skill_level=0
+ai.skill_level=7
 ai.is_barbarian=0
 gold=1540
 rates.tax=40
@@ -23281,7 +23281,7 @@ government_name="Monarchy"
 city_style_by_name="Babylonian"
 is_male=TRUE
 is_alive=TRUE
-ai.control=FALSE
+ai.control=TRUE
 diplstate={"type","max_state","first_contact_turn","turns_left","has_reason_to_cancel","contact_turns_left","embassy","gives_shared_vision"
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
@@ -23351,7 +23351,7 @@ ai={"spam","countdown","war_reason","patience","warn_space","ask_peace","ask_all
 2,-1,5,0,0,0,0,0
 3,-1,5,0,0,0,0,0
 }
-ai.skill_level=0
+ai.skill_level=7
 ai.is_barbarian=0
 gold=1720
 rates.tax=40
@@ -24864,7 +24864,7 @@ government_name="Monarchy"
 city_style_by_name="European"
 is_male=TRUE
 is_alive=TRUE
-ai.control=FALSE
+ai.control=TRUE
 diplstate={"type","max_state","first_contact_turn","turns_left","has_reason_to_cancel","contact_turns_left","embassy","gives_shared_vision"
 2,5,0,16,0,20,TRUE,FALSE
 4,4,0,0,0,20,TRUE,FALSE
@@ -24934,7 +24934,7 @@ ai={"spam","countdown","war_reason","patience","warn_space","ask_peace","ask_all
 3,-1,5,0,0,0,0,0
 4,-1,5,0,0,0,0,0
 }
-ai.skill_level=0
+ai.skill_level=7
 ai.is_barbarian=0
 gold=1430
 rates.tax=40
@@ -26436,7 +26436,7 @@ government_name="Monarchy"
 city_style_by_name="Asian"
 is_male=TRUE
 is_alive=TRUE
-ai.control=FALSE
+ai.control=TRUE
 diplstate={"type","max_state","first_contact_turn","turns_left","has_reason_to_cancel","contact_turns_left","embassy","gives_shared_vision"
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
@@ -26506,7 +26506,7 @@ ai={"spam","countdown","war_reason","patience","warn_space","ask_peace","ask_all
 4,-1,5,0,0,0,0,0
 0,-1,5,0,0,0,0,0
 }
-ai.skill_level=0
+ai.skill_level=7
 ai.is_barbarian=0
 gold=1550
 rates.tax=40
@@ -28021,7 +28021,7 @@ government_name="Monarchy"
 city_style_by_name="Tropical"
 is_male=TRUE
 is_alive=TRUE
-ai.control=FALSE
+ai.control=TRUE
 diplstate={"type","max_state","first_contact_turn","turns_left","has_reason_to_cancel","contact_turns_left","embassy","gives_shared_vision"
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
@@ -28091,7 +28091,7 @@ ai={"spam","countdown","war_reason","patience","warn_space","ask_peace","ask_all
 0,-1,5,0,0,0,0,0
 1,-1,5,0,0,0,0,0
 }
-ai.skill_level=0
+ai.skill_level=7
 ai.is_barbarian=0
 gold=1410
 rates.tax=40
@@ -29594,7 +29594,7 @@ government_name="Monarchy"
 city_style_by_name="European"
 is_male=TRUE
 is_alive=TRUE
-ai.control=FALSE
+ai.control=TRUE
 diplstate={"type","max_state","first_contact_turn","turns_left","has_reason_to_cancel","contact_turns_left","embassy","gives_shared_vision"
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
@@ -29664,7 +29664,7 @@ ai={"spam","countdown","war_reason","patience","warn_space","ask_peace","ask_all
 1,-1,5,0,0,0,0,0
 2,-1,5,0,0,0,0,0
 }
-ai.skill_level=0
+ai.skill_level=7
 ai.is_barbarian=0
 gold=1680
 rates.tax=40
@@ -31170,7 +31170,7 @@ government_name="Monarchy"
 city_style_by_name="Asian"
 is_male=TRUE
 is_alive=TRUE
-ai.control=FALSE
+ai.control=TRUE
 diplstate={"type","max_state","first_contact_turn","turns_left","has_reason_to_cancel","contact_turns_left","embassy","gives_shared_vision"
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
@@ -31240,7 +31240,7 @@ ai={"spam","countdown","war_reason","patience","warn_space","ask_peace","ask_all
 2,-1,5,0,0,0,0,0
 3,-1,5,0,0,0,0,0
 }
-ai.skill_level=0
+ai.skill_level=7
 ai.is_barbarian=0
 gold=1600
 rates.tax=40
@@ -32737,7 +32737,7 @@ government_name="Monarchy"
 city_style_by_name="European"
 is_male=TRUE
 is_alive=TRUE
-ai.control=FALSE
+ai.control=TRUE
 diplstate={"type","max_state","first_contact_turn","turns_left","has_reason_to_cancel","contact_turns_left","embassy","gives_shared_vision"
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
@@ -32807,7 +32807,7 @@ ai={"spam","countdown","war_reason","patience","warn_space","ask_peace","ask_all
 3,-1,5,0,0,0,0,0
 4,-1,5,0,0,0,0,0
 }
-ai.skill_level=0
+ai.skill_level=7
 ai.is_barbarian=0
 gold=1500
 rates.tax=40
@@ -34304,7 +34304,7 @@ government_name="Democracy"
 city_style_by_name="European"
 is_male=FALSE
 is_alive=TRUE
-ai.control=FALSE
+ai.control=TRUE
 diplstate={"type","max_state","first_contact_turn","turns_left","has_reason_to_cancel","contact_turns_left","embassy","gives_shared_vision"
 2,5,0,16,0,20,TRUE,FALSE
 2,5,0,16,0,20,TRUE,FALSE
@@ -34374,7 +34374,7 @@ ai={"spam","countdown","war_reason","patience","warn_space","ask_peace","ask_all
 4,-1,5,0,0,0,0,0
 0,-1,5,0,0,0,0,0
 }
-ai.skill_level=0
+ai.skill_level=7
 ai.is_barbarian=0
 gold=2190
 rates.tax=40
